tributors/main/orcid.py

```diff
@@ -129,7 +130,7 @@ def get_orcid_token():
     return orcid_token
 
 
-def record_search(url, email, interactive=False):
+def record_search(url, email, interactive=False, how=""):
```

Comment:
I can tell what "how" is based on reading below how the functions are using it - can we use a more descriptive label here, like search type, and document this in the docstring so it's clear the string is just for informational purposes and doesn't derive any logic?

Reply:
done in c76a90ad4e9ae51c9b4b07fc3ca0df62b1f2d147